icubam/predicu/tests/data/fake_all_bedcounts_2020-04-08_16h41.csv

```diff
@@ -1,101 +0,0 @@
-icu_name,date,datetime,icu_dept,n_covid_deaths,n_covid_healed,n_covid_transfered,n_covid_refused,n_covid_free,n_ncovid_free,n_covid_occ,n_ncovid_occ
```

Comment:
I would rather we re-generated this from the DB in memory in each test session, and made sure *.csv is in .gitignore to avoid commiting production dumps. Same comment for the .h5 below.

Otherwise we won't be sure that testing on this file is consistent with testing on production data.

We can do that, can't we?

Reply:
re-generate from which DB in memory? not sure i understand

Reply:
Well to run ICUBAM tests, one would run `scripts/populate_db_fake.py` https://github.com/icubam/icubam/blob/master/doc/install.md#pre-populate-db-with-test-data so we should be able to use that same data for our tests, without needing to rely on pre-generated CSV data.

That fake data generation script might need to be updated though. Not asking to do it here.